src/instance/key.txt

```diff
@@ -0,0 +1 @@
+uqxhV2luxBARhhBqVAFKpCkM0Az8H0Y5
```

Comment:
Remove from git repo (you can read/ask me why)

Reply:
I'll read it and will try to understand - will talk about this on today's meeting.